pkg/pb/piecestore2.proto

```diff
@@ -90,3 +90,23 @@ message RetainRequest {
 
 message RetainResponse {
 }
+
+// PieceHeader is used in piece storage to keep track of piece attributes.
+message PieceHeader {
+    // the storage format version being used for this piece. The piece filename should agree with this.
+    // The inclusion of this field is intended to aid repairability when filenames are damaged.
+    int32 format_version = 1;
```

Comment:
Should we define this as a protobuf enum to ensure an invalid `format_version` is not accidentally inserted?

Reply:
Oh, I didn't know protobuf actually checked that enum values were valid. That sounds like a great idea.